Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -1398,11 +1398,19 @@ FlowRelativeDirection RenderBox::physicalToFlowRelativeDirectionMapping(Physical
 {
     auto isHorizontalWritingMode = style().isHorizontalWritingMode();
     auto isLeftToRightDirection = style().isLeftToRightDirection();
+    // vertical-rl and horizontal-bt writing modes
+    auto isFlippedBlocksWritingMode = style().isFlippedBlocksWritingMode();
```

Comment:
this should be grabbed off of the ancestor (flex container, containing block etc) which sets up the writing mode for this box, see https://github.com/WebKit/WebKit/pull/11640#discussion_r1146464625

Reply:
I went ahead and added a lambda to get the style off of the formatting context. Currently it can grab it off of flexboxes and will `ASSERT_NOT_IMPLEMENTED_YET` for any other contexts. At the call site where `physicalToFlowRelativeDirection` is being called I added another lambda that checks to see if it can use this method so it doesn't trigger the assert